src/Catty/ViewController/Download/ProjectDetailStoreViewController.h

```diff
@@ -27,15 +27,16 @@
 #import "BaseTableViewController.h"
 
 @class CatrobatProject;
+@class StoreProjectDownloader;
+@class Data;
```

Comment:
Is this forward declaration really necessary?

Reply:
No, it is indeed not necessary. I've changed it!